tests/unit/commands/test_push.py

```diff
@@ -79,64 +77,6 @@ def test_push_a_snap(self):
         )
         mock_upload.assert_called_once_with("basic", self.snap_file)
 
-    def test_push_a_snap_running_from_snap(self):
```

Comment:
This test seems useful, is it duplicated somewhere?

Reply:
We clarified this offline: the tests covering get_tool_path here cover most of what is removed here. We need a suite that runs in docker to cover everything, which we can introduce later.